### PR TITLE
doc(RHTAPWATCH-820): Project* permissions

### PR DESCRIPTION
**Pull request by ifireball**
Update the roles and permissions ADR to include the permissions for the new Project Controller.

Should probably only be merged after #166 and redhat-appstudio/docs.appstudio.io#248 are merged.

### CODE REVIEW

**Review by amisstea — APPROVED** [review]
LGTM

File: ADR/0011-roles-and-permissions.md
Comment (arewm) [inline ADR/0011-roles-and-permissions.md:62]: Do you want to grant `delete` permissions to maintainers? Maintainers do not have those permissions for Components or Applications.

**Review by arewm — CHANGES_REQUESTED** [review]

File: ADR/0011-roles-and-permissions.md
Comment (ifireball) [reply ADR/0011-roles-and-permissions.md:62]: We did grant them in practice for now, so might as well have the docs reflect that. We can consider changing it later.

Comment (arewm) [reply ADR/0011-roles-and-permissions.md:62]: If that is the current state, would you mind including that update into this PR as well since it relates to the new permissions added?

Comment (ifireball) [reply ADR/0011-roles-and-permissions.md:62]: @arewm what changes should I make, if any?

Comment (ifireball) [reply ADR/0011-roles-and-permissions.md:62]: @arewm no, this PR reflects what the permissions for the project controller are ATM. Thos are, as you noted, slightly different the the permissions for Applications and Components

**Comment by ifireball** [comment]
@arewm can you move your review from "request changes" to "approved" if this does look good to you?

**Review by arewm — APPROVED** [review]
Approving as this is the current state. We should open a bug to remove the ability for maintainers to delete components and applications.

**Pull request merged**
